docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,177 @@
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation#id-prefix-reservation-criteria).
+> Reserving the prefix ID for your package will let you get the verified check mark:
+> ![image](media/Verified-check-mark.png)
+> Check out the [Package ID prefix reservation docs](https://docs.microsoft.com/en-us/nuget/nuget-org/id-prefix-reservation) to learn more.
+
+### Package Version
+
+✔️ CONSIDER using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
+> Essentially, this means using the Major.Minor.Patch[-prerelease] format.
+
+✔️ DO include a pre-release suffix when releasing a non-stable package.
+
+See the [.NET library versioning guide](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/versioning) for more advanced guidance including aseembly versioning.
+
+### Authors
+
+✔️ CONSIDER including the current NuGet.org owners for consistency across NuGet clients.
+> Visual Studio currently displayed `Authors` on the Browse tab. It is also in the criteria for ID prefix reservation.
+
+### Description
+
+✔️ DO include a short description for your package that describes what it's for.
+> Package descriptions are one of the most prominent fields surfaced in NuGet search and will likely be the first thing a potential consumers looks at to determine if a package is right for them.
+
+### Copyright
+
+✔️ CONSIDER copyrighting your package with "Copyright (c) <name/company\> <year\>."
+>A copyright notice essentially indicates that your work cannot be copied without your permission. Including, a copyright notice in your package is easy and won't do any harm!
+
+Example: Copyright (c) Contoso 2020
+
+### Licensing
+
+✔️ DO specify a valid license for your package.
+> [!IMPORTANT]
+> A project without a license defaults to [exclusive copyright](https://choosealicense.com/no-permission/), making it legally impossible for other people to use.
+
+✔️ CONSIDER using a license expression (SPDX identifier).
+> License expressions are surfaced the most clearly to package consumers and make it more obvious if the license has changed with a new release.
+
+✔️ CONSIDER specifying an MIT license.
+> If you want your package as to be usable by as many consumers as possible, the MIT license is a very well known non-restrictive license.
+
+❌ DO NOT use the deprecated `LicenseUrl` metadataproperty.
+> This presents legal ambiguity as license changes at the URL will retroactively change the displayed license for previous package versions.
+
+### Project URL
+
+✔️ CONSIDER including a link to an associated project or company website.
+> Company website links, even if not specific to the package/project, still improve consumer trust.
+
+❌ DO NOT put a repository URL (GitHub, BitBucket, etc.) in the `Project URL` field, as consumers will expect to find it in the `Repository URL` field.
```

Comment:
I would make this an "Avoid" not a "DO NOT". Do we have confidence that repository URL has good support on all tooling (both 1st and 3rd party)?

Reply:
Could you elaborate a bit further? Any specific tools you're worried about?

Reply:
Well, VS PM UI does not show repository URL, IIRC. And for many small open source projects the project URL _is_ the repository URL.

Reply:
TIL PMUI doesn't show repo URL. Why don't we?

Reply:
The work was never done. The metadata is not plumbed through to all of the API endpoints that need it (package metadata resource, i.e. registration) and the client work was not done.
https://github.com/NuGet/Home/issues/8584